infra/modules/database/role_manager/role_manager.py

```diff
@@ -2,9 +2,14 @@
 import itertools
 from operator import itemgetter
 import os
+import json
```

Comment:
via @chouinar and https://github.com/navapbc/template-infra/blame/7f43aaa9f1ee630e4d674d3197bbcaf134bafc19/infra/modules/database/role_manager/role_manager.py#L5